src/Extensions/TimedScopes/TimedScopeExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.Omex.Extensions.Logging.TimedScopes
+{
+	/// <summary>
+	/// Extensions for TimedScopeResult enum
+	/// </summary>
+	public static class TimedScopeExtensions
+	{
+		/// <summary>
+		/// Decides whether the result is success
+		/// </summary>
+		/// <param name="result">The result</param>
+		/// <returns>Success flag (null if we don't know)</returns>
+		public static bool? IsSuccessful(this TimedScopeResult result) =>
+			result switch
+			{
+				default(TimedScopeResult) => null,
+				TimedScopeResult.Success => true,
+				_ => false,
+			};
+
+
+		/// <summary>
+		/// Decides whether we should replay events for scopes with given result
+		/// </summary>
+		/// <param name="result">The result</param>
+		/// <returns>true if we should replay events for this result; false otherwise</returns>
+		public static bool ShouldReplayEvents(this TimedScopeResult result) =>
+			result switch
+			{
+				TimedScopeResult.SystemError => true,
+				_ => false,
+			};
+	}
+
+	/// <summary>
+	/// Extensions for Activity
+	/// </summary>
+	public static class ActivityExtensions
```

Comment:
>ActivityExtensions [](start = 21, length = 18)

This should be in a different file entitled `ActivityExtensions.cs`

Reply:
Moved, thanks